### ISSUE TITLE
Outputs always converge to wrong values

### ISSUE THREAD

**Issue by georgenardes** [JY-112553/yolov1-keras-voc#3]
Hi everyone,

I have trained for more than 50 epochs with one single image and the output box doesn't get closer to the ground truth.
I think that it should overfit the data, but it is converging to a wrong value. I put a "K.print_tensor(y_pred[0, 3, 3])" in the loss function to see what is happening and the same values are predicted in all tries. The index [0,3,3] is the grid in which the object center is contained and should get closer to the y_true[0,3,3] values.
Has anyone got trough this like me?
